test/domains/userAPI/isEmptyTestDomain.good

```diff
@@ -0,0 +1,5 @@
+false
+true
+false
+true
+
```

Comment:
This extra line will cause the test to fail. You can check this on your end with:
```
start_test isEmptyTestDomain.chpl
```

Reply:
ok